app/src/main/java/com/example/evently/ui/common/EventDetailsFragment.java

```diff
@@ -78,20 +78,20 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                     String.valueOf(eventEntrants.all().size()));
         });
 
-        if (savedInstanceState == null) {
-            // Load the entrants list fragment if we were not recreated.
-            getChildFragmentManager()
-                    .beginTransaction()
-                    .setReorderingAllowed(true)
-                    .add(R.id.entrantListContainer, getFragmentForEntrantListContainer(), null)
-                    .commit();
-            // Also load the action buttons fragment.
-            getChildFragmentManager()
-                    .beginTransaction()
-                    .setReorderingAllowed(true)
-                    .add(R.id.actionButtonsContainer, getFragmentForActionButtonsContainer(), null)
-                    .commit();
-        }
+        if (savedInstanceState != null) return;
+
+        // Load the entrants list fragment if we were not recreated.
```

Comment:
Should move the comment before the "if -> return". This restructure makes it easy for future additions to just get skipped accidentally

i.e someone wants to add some more stuff to this method that doesn't depend on `savedInstanceState` being `null`. They might naturally add it to the end of the method. The comment above the return could at least shed light on "we're only early returning because there's nothing happening here other than fragment spawns"